docs/how-to-guides/config-settings.rst

```diff
@@ -9,41 +9,53 @@ Use build config settings
 *************************
 
 Build config settings are settings you can pass to ``meson-python`` when
-building your project. They can be use to customize the build in some aspect.
+building your project. They can be use to customize the build in some
+aspect.
 
-The way you specify them depends on which tool you are using, here are the most
-popular ones.
+Several Python build front-ends exist, with different ways to pass
+configuration settings to the build back-end. The most popular are
+``pypa/build``, which uses the ``-C`` command line option, and ``pip``,
+which uses the ``--config-settings`` option. For example:
 
+.. tab-set::
 
-`pypa/build`_
-=============
+    .. tab-item:: pypa/buid
+        :sync: key_pypa_build
 
-On `pypa/build`_, they can be used via the ``-C``/``--config-setting`` option.
+	.. code-block:: console
 
+	   $ python -m build \
+               -Csetup-args="-Doption=true" \
+               -Csetup-args="-Dvalue=1" \
+               -Ccompile-args="-j6"
 
-.. code-block:: console
+    .. tab-item:: pip
+        :sync: key_pip
 
-   $ python -m build -Ckey=value
+	.. code-block:: console
 
+	   $ python -m pip wheel . \
+               --config-settings=setup-args="-Doption=disable" \
+               --config-settings=compile-args="-j6"
 
-Please check the `pypa/build documentation`_ for more information.
+This examples use the ``python -m pip wheel`` command to build a Python
+wheel that can be later installed or distributed. To build a package and
+immediately install it, just replace ``wheel`` with ``install``.
```

Comment:
I think this should be an admonition, also IMO the `python -m` bit of the command can be dropped in this text.

typo: "examples" > "example"

```suggestion
.. admonition:: Using ``pip install``
   :class: hint

   This example uses the ``pip wheel`` command to build a Python
   wheel that can be later installed or distributed. To build a package and
   immediately install it, just replace ``wheel`` with ``install``.
```

Reply:
The command executed is `python -m pip wheel`. Wheel is (was?) a command from the `wheel` package. Why should this an admonition? There is no reason to focus the attention to the reader to this paragraph, it is just a pretty boring explanation of what the command does. I'm not a fan of using heavy markup for no reason.